vendor/wp-now/src/wp-now.ts

```diff
@@ -158,6 +129,55 @@ export default async function startWPNow(
 	};
 }
 
+async function getPHPInstance( options: WPNowOptions, isPrimary: boolean, requestHandler: PHPRequestHandler ) : Promise<{ php: PHP; runtimeId: number }> {
+	const id = await loadNodeRuntime( options.phpVersion );
+	const php = new PHP(id);
+
+	await setPhpIniEntries(php, {
+		'memory_limit': '256M',
+		'disable_functions': '',
+		'allow_url_fopen': '1',
+		'openssl.cafile': '/internal/shared/ca-bundle.crt'
+	});
+
+	return { php, runtimeId: id };
+}
+
+function prepareDocumentRoot( php: PHP, options: WPNowOptions ) {
+	php.mkdir(options.documentRoot);
+	php.chdir(options.documentRoot);
+	php.writeFile(
+		`${options.documentRoot}/index.php`,
+		`<?php echo 'Hello wp-now!';`
+	);
```

Comment:
Unrelated to this PR but I wonder if this is really needed 🤔 .

Reply:
I wasn't sure of the purpose so for now i left it. I think this whole file could use some refactoring as there is a lot of logic in there that does not apply to Studio. Its a bit difficult to know what is there for a reason and what is there because it was once copied from wp-now.

Reply:
Yeah, I agree. Let's keep it as-is for now.